mlos_bench/mlos_bench/launcher.py

```diff
@@ -257,8 +257,8 @@ def _expand_vars(self, value: Any) -> Any:
             # new set of params explicitly.
             # Since this operates by updating the global_config along the way,
             # we need to continually update the set of params to use for substitution.
-            params = dict(os.environ)
-            params.update(self.global_config)
+            params = self.global_config.copy()
+            params.update(dict(os.environ))
```

Comment:
```suggestion
            params.update(os.environ)
```
or is it there to make mypy happy?

Reply:
Yeah, it's a mypy thing.

`os.Environ` as a type of `EnvironVars` or something like that instead of a `dict`